Automattic/wp-calypso
# Fill input with state data

## Proposed Changes


Read value from store on setup step and fill the inputs, it will help when navigating back/forward the steps


## Test Instructions

- Pull this branch and run `yarn start` from the root folder
- Navigate to `setup/intro?flow=link-in-bio or setup/intro?flow=newsletter`
- Advance further on the flow until the domain step
- Navigate back using the browser back button
- Check if the inputs are filled with data


This PR needs to be merged before or together with [this](https://github.com/Automattic/wp-calypso/pull/67252)
